contracts/Comptroller/ComptrollerStorage.sol

```diff
@@ -234,3 +234,21 @@ contract ComptrollerV11Storage is ComptrollerV10Storage {
     //mapping(address borrower => mapping (address delegate => bool approved)) public approvedDelegates;
     mapping(address => mapping(address => bool)) public approvedDelegates;
 }
+
+contract ComptrollerV12Storage is ComptrollerV11Storage {
+    struct FacetAddressAndPosition {
+        address facetAddress;
```

Comment:
Why don't we use `facetAddressPosition`, as we are doing in `FacetFunctionSelectors`